js/interface.js

```diff
@@ -994,9 +999,25 @@ var DynamicLists = (function() {
           $('.layout-holder[data-layout="' + _this.config.layout + '"]').addClass('active');
 
           // Load Add. Edit, Delete
-          $('#add_entry').prop('checked', _this.config.addEntry).trigger('change');
-          $('#edit_entry').prop('checked', _this.config.editEntry).trigger('change');
-          $('#delete_entry').prop('checked', _this.config.deleteEntry).trigger('change');
+          $('#add_entry')
+            .prop('checked', _this.config.addEntry)
+            .parent()
+            .find('.hidden-settings')
+            .toggleClass('active', !!_this.config.addEntry);
```

Comment:
It doesn't have to be a boolean.

```suggestion
            .toggleClass('active', _this.config.addEntry);
```

Reply:
> It doesn't have to be a boolean.

In our case _this.config.addEntry can be undefined and .toggleClass does not have any type conversions inside that's why we must convert it to boolean.

Reply:
@OleksandrLavrynenko Yes you're right. I just made the same mistake in another code that I was working on. We do need to convert it to boolean 👍🏻  Good work!